src/Workspaces/Core/Portable/Remote/PinnedSolutionInfo.cs

```diff
@@ -18,7 +18,7 @@ internal sealed class PinnedSolutionInfo
         /// This later used to find matching solution between VS and remote host
         /// </summary>
         [DataMember(Order = 0)]
-        public readonly int ScopeId;
+        public readonly Checksum SolutionChecksum;
```

Comment:
note: we already had this.  it just moved up as being the actual fully unique identifier.